rules/integrations/aws/execution_ssm_command_document_created_by_rare_user.toml

```diff
@@ -0,0 +1,97 @@
+- **Check Document Status and Metadata**:
+  - **Document Status**: Confirm the document creation status in `aws.cloudtrail.response_elements.documentDescription.status`. A status of `Creating` may indicate that the document is in progress.
+  - **Execution Permissions**: Review if the document specifies `platformTypes` and `documentVersion` in `aws.cloudtrail.response_elements.documentDescription` to understand which environments may be impacted and if multiple versions exist.
+
+### False Positive Analysis
+
+- **Authorized Administrative Actions**: Determine if this document creation aligns with scheduled administrative tasks or actions by authorized personnel.
+- **Historical User Actions**: Compare this action against historical activities for the user to determine if they have a history of creating similar documents, which may indicate legitimate usage.
+
+### Response and Remediation
+
+- **Immediate Document Review and Deletion**: If the document creation is deemed unauthorized, delete the document immediately and check for other similar documents created recently.
+- **Enhance Monitoring and Alerts**: Configure additional monitoring for SSM document creation events, especially when associated with untrusted or rare users.
+- **Policy Update**: Consider restricting SSM document creation permissions to specific, trusted roles or users to prevent unauthorized document creation.
+- **Incident Response**: If the document is confirmed as part of malicious activity, treat this as a security incident. Follow incident response protocols, including containment, investigation, and remediation.
+
+### Additional Information
+
+For further guidance on managing and securing AWS Systems Manager in your environment, refer to the [AWS SSM documentation](https://docs.aws.amazon.com/systems-manager/latest/userguide/what-is-systems-manager.html) and AWS security best practices.
+"""
+references = [
+    "https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_CreateDocument.html",
+    "https://docs.aws.amazon.com/systems-manager/latest/userguide/documents.html"
+]
+risk_score = 21
+rule_id = "50a2bdea-9876-11ef-89db-f661ea17fbcd"
+severity = "low"
+tags = [
+    "Domain: Cloud",
+    "Data Source: AWS",
+    "Data Source: Amazon Web Services",
+    "Data Source: AWS SNS",
+    "Data Source: AWS Systems Manager",
+    "Resources: Investigation Guide",
+    "Use Case: Threat Detection",
+    "Tactic: Execution"
+]
+timestamp_override = "event.ingested"
+type = "new_terms"
+
+query = '''
+event.dataset: "aws.cloudtrail"
+    and event.provider: "ssm.amazonaws.com"
+    and event.action: "CreateDocument"
+    and event.outcome: "success"
+    and aws.cloudtrail.response_elements: *documentType=Command*
```

Comment:
why this distinction?

Reply:
https://docs.aws.amazon.com/systems-manager/latest/userguide/documents.html

Command documents are the focus here to capture RunShellScript RunPowerShellScript or custom documents that are meant to run system commands upon detonation.